### PR TITLE
Add JioSaavn testimonial

### PR DESCRIPTION
**Pull request by bitsondatadev**

### CODE REVIEW

File: _data/users.yml [hunk 1]
```diff
@@ -163,6 +163,23 @@
...
+  logo: /assets/images/logos/jiosaavn.png
+  logosmall: /assets/images/logos/jiosaavn-small.png
+  testimonial: |
+    At JioSaavn, Trino plays a pivotal role in the data engineering stack and is
+    the central integrated hub for all the data needs. It is used in ETL
+    processing since its performance is much better than Hive and Spark for
+    certain use cases. Trino analyzes billions of events logged everyday and
+    generates insights with adhoc analytical queries. It is also very useful to
+    query data from various databases like MongoDB, Cassandra, and MySQL. Trino
+    has is also extensively used to power BI tools and dashboards from both ETL
```
Comment (hashhar) [inline _data/users.yml:178]: `Trino has` -> `Trino here` I assume?

File: _data/users.yml [hunk 2]
```diff
@@ -163,6 +163,23 @@
...
+    pipeline data in ORC and Parquet files and real-time data using Pinot Druid
```
Comment (hashhar) [inline _data/users.yml:179]: Pinot and Druid?

**Comment by martint** [comment]
Can you remove some of the white space around the logos so that they don't appear too small compared to other logos? Take a look at how other logos are formatted.

**Comment by bitsondatadev** [comment]
> Can you remove some of the white space around the logos so that they don't appear too small compared to other logos? Take a look at how other logos are formatted.

Done!

**Pull request merged**